source/application/CLS/CLSAppController.cpp

```diff
@@ -83,6 +85,24 @@ void CLSAppController::shutdown()
 }
 
 // ============== implementation of protected slots =====================
+void CLSAppController::onUserConfigurationLoadedFromDb()
+{
+	AMXRFDetector *xrfDetector = CLSBeamline::clsBeamline()->xrfDetector();
```

Comment:
in AMBeamline, we have A function `addSynchronizedXRFDetector()`, which can help synchronize the settings among the XRF detectors.